exercises/perfect-numbers/src/test/scala/PerfectNumbersTest.scala

```diff
@@ -0,0 +1,29 @@
+import org.scalacheck.Gen
+import org.scalatest.prop.PropertyChecks
+import org.scalatest.{Matchers, PropSpec}
+
+class PerfectNumbersTest extends PropSpec with Matchers with PropertyChecks {
+
+  property("deficient numbers") {
+    val deficientVals = Gen.oneOf(3, 7, 15, 33550337)
```

Comment:
This is not a good case for Property-based Testing (PPT). Better keep the conventional tests that you had before.

For PPT you would define general "properties" (or "laws") that always hold. This is called [Generator-driven property checks](http://www.scalatest.org/user_guide/generator_driven_property_checks) in ScalaTest terminology.

One example that comes to my mind: "For all `Int`s: a number is always either deficient, perfect or abundant".
You could add this test to the conventional tests if you want.